### DISCUSSION TITLE
The Weight of Unsharing

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#400] [Announcements]
### Bug fixe(s)
* Fix share-to-vanilla in config.json (@blu-dev)
* Fix infinite loadings (@blu-dev)

### Note
* If you go online with added slots, you take full responsibility on whatever happens to your Switch or Nintendo Online account.
* We do not endorse going online with added slots and we are only trying to make it more safe to not risk running into trouble with Nintendo. If you get banned, that's on you. If you get disconnected, that's also on you. Choose carefully.

## Donations
I'd like to personally thank every ARCropolis contributor for putting up the selfless and thankless job of implementing new features and fixing bugs.
As you may know, ARCropolis is a **free** modloader built during our spare time, as all of us have to make a living while sacrificing our time to the cause.
If you could spare to make a donation, or support our other activities, we would be extremely grateful.

## Raytwo (@Raytwo)
* [Ko-fi](https://ko-fi.com/raytwo)
* [Twitter](https://twitter.com/Raytwo_ssb)

## blujay (@blu-dev)
* [Ko-fi](https://ko-fi.com/bludev)
* [Patreon](https://www.patreon.com/thebludev)
* [Twitter](https://twitter.com/jayblu_)

## CoolSonicKirby (@Coolsonickirby)
* [Ko-fi](https://ko-fi.com/coolsonickirby)
* [Twitter](https://twitter.com/coolsonickirby2)

## jozz (@jozz024)
* [Twitter](twitter.com/jozz02420)

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.6.2'>The Weight of Unsharing</a>.</em>

**Comment by natecotto71** [comment]
me and a few people have been having this issue, but after downloading the update for acropolis, my game consistently crashes at launch. any ideas on what the issue might be?

**Reply by Lagmobile** [reply]
I have also been crashing since 3.5.1. Downgrading to 3.5.0 seems to be the only fix for now.

**Comment by Ryen-Orco** [comment]
This definitely fixed an issue I was having with Marth and the one-slot effects plugin (plugin would panic and say the config.json was incorrect, but with this it was fine and worked). A new issue I'm having though is that the effects work for a while, but after a few battles all effects not used prior do not load (including default effects), and it eventually crashes ultimate with a freeze, with the default switch error screen eventually popping up. Is this an issue with arcropolis or the plugin? Other than that, this is really cool and you guys are amazing on how far you can mod smash!

**Comment by MikeyProHD** [comment]
I tested this spoofing mod using a Mr. L mod that was set to go over the custom c08 slot (Link to the mod here: https://t.co/16aDUg0B0W). In training mode with two Luigi's present, I discovered that whether or not you're player 1 with the mod enabled, the CPU Luigi won't play any voice lines. There are also no sound effects relating to rolling, spot dodging, jumping (The double jump sound effect plays, though), no walking or running beside the universal stage walking sound effect, no crouching nothing for his final smash, etc. The hit sound effects seem to work fine, though.
